content/01.abstract.md

```diff
@@ -1,3 +1,18 @@
 ## Abstract {.page_break_before}
 
-
+Scientific news coverage shapes the public’s view of the current state of scientific findings and proposes legitimate experts.
+Through researching a story, journalists identify and interview a limited number of sources.
+These sources may come from a journalist’s research or through recommendations by other scientists.
+In both cases, unconscious biases may influence who is identified as an expert to interview, possibly skewing the selection of interviewees.
```

Comment:
```suggestion
In either case, unconscious biases may influence who is identified as an expert to interview, possibly skewing the selection of interviewees.
```

Reply:
this _felt_ like an either situation to me but I'm not sure - this is a soft recommendation